auth/jwt/jwt.go

```diff
@@ -80,84 +71,44 @@ func (j *jwt) Generate(id string, opts ...auth.GenerateOption) (*auth.Account, e
 	return account, nil
 }
 
-func (j *jwt) Grant(role string, res *auth.Resource) error {
+func (j *jwt) Grant(rule *auth.Rule) error {
 	j.Lock()
 	defer j.Unlock()
-	j.rules = append(j.rules, &rule{role, res})
+	j.rules = append(j.rules, rule)
```

Comment:
Should we check for dupes by checking ID?

Reply:
Since the JWT isn't backed by the store, the rules are normally hardcoded in the service (like we do in micro/micro/internal/auth), so I don't think this is really necessary.